index.html

```diff
@@ -2,10 +2,25 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./style.css" />
     <script src="index.js" defer></script>
     <title>Black Jack Game</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app" class="container">
+      <a href="#" class="shuffle">Shuffle</a>
```

Comment:
This should be a button since it's an interactive element and not a navigation action, anchor are only for navigation